package.json

```diff
@@ -792,21 +792,21 @@
         "ts-node": "^7.0.1",
         "typescript": "^4.3.5",
         "vsce": "^2.11.0",
+        "vscode-azurekudu": "^0.2.3",
```

Comment:
I think you can just remove this entirely. We are not using the types anywhere in this code.